models/DataObject/AbstractObject.php

```diff
@@ -171,6 +172,11 @@ class AbstractObject extends Model\Element\AbstractElement
      */
     protected $o_childrenSortBy;
 
+    /**
+     * @var bool
+     */
+    protected $o_reverseSort;
```

Comment:
please change property name to make it more clear and somehow consistent with o_childrenSortBy....maybe "o_childrenSortDesc" or "o_childrenSortOrder" with values 'ASC/DESC'